INFO     cardano_node_tests.tests.conftest:conftest.py:135 Changed CWD to '/home/martink/Source/repos/cardano-node-tests2/tmp/pytest-of-martink/pytest-306/popen-gw2'.
INFO     cardano_node_tests.utils.cluster_management:cluster_management.py:437 Stopping cluster with `/home/martink/Source/repos/cardano-node-tests2/tmp/pytest-of-martink/pytest-306/cluster1/startup_files/oiumgtwy/stop-cluster-hfc`.
INFO     cardano_node_tests.utils.artifacts:artifacts.py:40 Start script coverage log file saved to '/home/martink/Source/repos/cardano-node-tests2/.cli_coverage/cli_coverage_script_220622_001930308_ndnj.log'.
INFO     cardano_node_tests.utils.artifacts:artifacts.py:64 Cluster artifacts saved to '/home/martink/Source/repos/cardano-node-tests2/tmp/pytest-of-martink/pytest-306/cluster_artifacts/state-cluster1_ankabvfh'.
INFO     cardano_node_tests.utils.cluster_nodes:cluster_nodes.py:340 Starting cluster with `/home/martink/Source/repos/cardano-node-tests2/tmp/pytest-of-martink/pytest-306/cluster1/startup_files/oiumgtwy/start-cluster-hfc`.
INFO     cardano_node_tests.utils.cluster_nodes:cluster_nodes.py:342 Cluster started.
INFO     cardano_node_tests.utils.cluster_nodes:cluster_nodes.py:364 Performing 'restart' action on services ['nodes:pool2', 'nodes:pool3'].
INFO     cardano_node_tests.tests.test_kes:test_kes.py:204 2022-06-22 00:43:32.439529: Waiting for 200 sec for KES expiration.
INFO     cardano_node_tests.tests.test_kes:test_kes.py:208 2022-06-22 00:46:52.538306: KES expired (?); tip: '{'block': 725, 'epoch': 8, 'era': 'Babbage', 'hash': 'a3c26f0852c091dfd6a402f540e3a26bd7e2d8a3baabd8cbc9d2873f1fefc87a', 'slot': 7285, 'syncProgress': '100.00'}'.
INFO     cardano_node_tests.tests.test_kes:test_kes.py:117 2022-06-22 00:46:52.576216: Waiting for 1 new epoch(s).
INFO     cardano_clusterlib.clusterlib_klass:clusterlib_klass.py:3028 Waiting for 164.00 sec for slot no 8105.
INFO     cardano_node_tests.tests.test_kes:test_kes.py:120 2022-06-22 00:49:36.736171: Waiting for the end of current epoch.
INFO     cardano_node_tests.utils.cluster_nodes:cluster_nodes.py:364 Performing 'restart' action on services ['nodes:pool2', 'nodes:pool3'].
INFO     cardano_node_tests.tests.test_kes:test_kes.py:225 2022-06-22 00:52:43.429756: Waiting 120 secs to make sure the expected errors make it to log files.